internal/pkg/skuba/kured/kured.go

```diff
@@ -44,7 +44,7 @@ func Lock(client clientset.Interface) error {
 	if err != nil {
 		return errors.Wrap(err, "unable to patch daemonset with kured locking annotation")
 	}
-	klog.V(1).Info("successfully annotated daemonset with kured locking annotation")
+	klog.V(2).Info("successfully annotated daemonset with kured locking annotation")
```

Comment:
I wasn't clear on the frequency or importance of the log messages in kured.go.  If they are rarely logged and of high importance, then leaving them at V(1) may be appropriate.

Reply:
IMO 2 is fine here.